pelican/tests/output/basic/a-markdown-powered-article.html

```diff
@@ -9,7 +9,7 @@
 
 <body id="index" class="home">
         <header id="banner" class="body">
-                <h1><a href="/">A Pelican Blog </a></h1>
```

Comment:
This is a mysterious change.

Reply:
Agreed. It makes one wonder why the space was there in the first place, no? Perhaps a bug that was fixed?

Reply:
It seems that Jinja v2.11+ is what changed the handling of this particular space.